packages/flutter/lib/src/widgets/dismissible.dart

```diff
@@ -62,6 +62,8 @@ enum DismissDirection {
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=iEMgjrfuc58}
 ///
+/// If [direction] is null, then the widget will not be dismissible.
```

Comment:
This feels like a strange API.

Reply:
I think the original author intended for the direction being null to serve as a flag for whether the Dismissible is active? Maybe adding a flag specific to that purpose would be more appropriate, like `active: true` or something similar.

Reply:
I think we shouldn't provide this functionality at all (especially since it adds surprising restrictions). If you don't want it to be dismissible, don't use a dismissible. If you're worried about duplicating code, write a helper widget that encapsulates the duplication.